config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -72,6 +72,44 @@ presubmits:
           requests:
             cpu: 8
             memory: 20Gi
+  - name: pull-kubernetes-integration-race
+    cluster: eks-prow-build-cluster
+    always_run: false # Has known failures.
+    decorate: true
+    skip_branches:
+    - release-\d+.\d+ # per-release job
+    annotations:
+      fork-per-release: "true"
+      testgrid-num-failures-to-alert: '10'
+      testgrid-alert-stale-results-hours: '24'
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: sig-testing-canaries
+    path_alias: k8s.io/kubernetes
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250527-1b2b10e804-master
+        command:
+        - runner.sh
+        env:
+        - name: KUBE_TIMEOUT
+          value: "-timeout=1h30m"
+        - name: KUBE_RACE
+          value: "-race"
```

Comment:
This doesn't work yet: test-integration.sh unconditionally sets it to `""`. The job can be used to test a PR which changes that...